rotkehlchen/chain/evm/decoding/aave/common.py

```diff
@@ -72,6 +73,22 @@ def decode_liquidation(self, context: DecoderContext) -> None:
         token and part of the collateral deposited is lost too. Those two events happen as
         transfers in a transaction started by the liquidator."""
 
+    def _is_aave_token_contract(self, queried_address: 'ChecksumEvmAddress | None') -> bool:
+        """Utility function for checking if a queried_address is an aave token.
+        Returns True if it is an aave token, or False if it is not."""
```

Comment:
where it is used you have already the token sometimes. Make it two function. `_address_is_aave_contract` and `_token_is_aave_contract`